apps/yapms/src/routes/app/+layout.svelte

```diff
@@ -0,0 +1,47 @@
+<script>
+	import '$lib/styles/global.css';
+	import { goto } from '$app/navigation';
+	import { page } from '$app/stores';
+	import { LoadedMapStore } from '$lib/stores/LoadedMap';
+	import { SavedMapSchema } from '$lib/types/SavedMap';
+	import { onMount } from 'svelte';
+	import { LoadingErrorModalStore } from '$lib/stores/Modals';
+
+	// this layout will will redirect the user to the proper map
+	// if they specify the m query parameter
+	onMount(async () => {
+		const map = encodeURIComponent($page.url.searchParams.get("m") ?? "");
+		// if the user is not loading a map, redirect them to the default map
+		if (map === "") {
+			await goto(`/app/usa/presidential/2022`);
+			return;
+		}
+
+		// load the requested map from pocketbase
+		const data = await fetch(`http://localhost:8080/api/files/maps/${map}/data.json.gz`);
```

Comment:
Same thing here with the environment variable

Reply:
Fixed